server/server.js

```diff
@@ -12,8 +15,43 @@ var corsOptions = {
   optionsSuccessStatus: 200, // some legacy browsers (IE11, various SmartTVs) choke on 204
 };
 
+const api = bizSdk.FacebookAdsApi.init(process.env.ACCESS_TOKEN);
+const showDebugingInfo = true; // Setting this to true shows more debugging info.
+if (showDebugingInfo) {
+  api.setDebug(true);
+}
+
+let url = null;
+const logApiCallResult = (apiCallName, data) => {
+  console.log(apiCallName);
+  url = apiCallName;
+  if (showDebugingInfo) {
+    console.log('Data:' + JSON.stringify(data));
+  }
+};
+
+let fields, params;
+fields = [
+];
+params = {
+};
+
+let sample_code, field_data, response;
+module.exports.getRawValue = async function getRawValue(){
+  sample_code = await (new Lead(process.env.LEAD_ID)).get(
```

Comment:
You should scrape the lead ID from the URL and pass it from frontend, instead of reading from env file